### DISCUSSION TITLE
ios swift build failed

### DISCUSSION THREAD

**Discussion by enchhuang** [trustwallet/wallet-core#2770] [Q&A]
Failed to build module 'WalletCore'; this SDK is not supported by the compiler (the SDK is built with 'Apple Swift version 5.7 (swiftlang-5.7.0.127.4 clang-1400.0.29.50)', while this compiler is 'Apple Swift version 5.7.1 (swiftlang-5.7.1.135.3 clang-1400.0.29.51)'). Please select a toolchain which matches the SDK.

**Comment by hewigovens** [comment]
Try to clean and rebuild

**Comment by henadzibahdanovich** [comment]
check this
maybe this will help you
https://stackoverflow.com/questions/70556401/swift-version-conflict-this-sdk-is-not-supported-by-the-compiler-please-select
